content/doc/upgrade-guide/2.138.adoc

```diff
@@ -0,0 +1,43 @@
+---
+layout: documentation
+title:  Jenkins Upgrade Guide
+notitle: true
+---
+
+== Upgrading to Jenkins LTS 2.138.x
+
+Each section covers the upgrade from the previous LTS release, the section on 2.138.1 covers the upgrade from 2.121.3.
+
+=== Upgrading to Jenkins LTS 2.138.1
+
+==== New login and user signup pages
+
+link:/blog/2018/06/27/new-login-page/[Announcement blog post],
+https://issues.jenkins-ci.org/browse/JENKINS-50447[JENKINS-50447]
+
+The login and user signup pages have been redesigned.
+As a side effect, existing `PageDecorator` implementations will not be used on the redesigned pages.
+
+See the link:/blog/2018/06/27/new-login-page/[announcement blog post] for further information.
+
+
+==== New API token system
+
+link:/blog/2018/07/02/new-api-token-system/[Announcement blog post],
+https://issues.jenkins-ci.org/browse/JENKINS-32442[JENKINS-32442],
+https://issues.jenkins-ci.org/browse/JENKINS-32776[JENKINS-32776]
+
+The per-user API tokens that allow access to the HTTP remote API have been redesigned:
+API tokens can now be created and revoked, and are stored in a non-recoverable format.
+
+See the link:/blog/2018/06/27/new-login-page/[announcement blog post] for further information.
+
+
+==== Disabled deprecated agent protocols
+
+https://issues.jenkins-ci.org/browse/JENKINS-48480[JENKINS-48480]
+
+The deprecated Jenkins CLI Protocol versions 1 and 2, and Java Web Start Agent Protocol versions 1, 2, and 3 have been disabled.
```

Comment:
"... disabled on all instances without explicit enable flags / settings in configuration-as-code". I would reference the announcement about disabling for new installations. https://jenkins.io/blog/2017/08/11/remoting-update/

Reply:
Which is basically the same thing as 're-enable after upgrade' as below.

I'm really not sure what to do about this.

And as my confused reaction earlier may have indicated, I'm not sure the blog post is a good reference, especially as the only blog post referenced on this page that's not directly related to the change actually applied here.

For now I'll keep it as is, and we can extend in a followup change.